src/components/PlantingSites/PlantingSitesList.tsx

```diff
@@ -3,17 +3,22 @@ import { Box, CircularProgress, Grid, Typography } from '@mui/material';
 import { Button, theme } from '@terraware/web-components';
 import { useDeviceInfo } from '@terraware/web-components/utils';
 import { TrackingService } from 'src/services';
-import { OrNodePayload, SearchResponseElement, SearchSortOrder } from 'src/types/Search';
+import { SearchNodePayload, SearchResponseElement, SearchSortOrder } from 'src/types/Search';
 import strings from 'src/strings';
 import useDebounce from 'src/utils/useDebounce';
 import PageSnackbar from 'src/components/PageSnackbar';
 import PageHeaderWrapper from 'src/components/common/PageHeaderWrapper';
 import TfMain from 'src/components/common/TfMain';
 import EmptyStatePage from 'src/components/emptyStatePages/EmptyStatePage';
-import PlantingSitesTable from './PlantingSitesTable';
-import PlantingSiteTypeSelect from './PlantingSiteTypeSelect';
+import PlantingSitesTable from 'src/components/PlantingSites/PlantingSitesTable';
+import PlantingSiteTypeSelect from 'src/components/PlantingSites/PlantingSiteTypeSelect';
 import { useOrganization, useTimeZones } from 'src/providers/hooks';
 import { setTimeZone, useDefaultTimeZone } from 'src/utils/useTimeZoneUtils';
+import useForm from 'src/utils/useForm';
+
+export type PlantingSitesFilters = {
```

Comment:
thoughts on moving this to src/types/PlantingSites or some other common file.. only because we have PlantingSitesTable importing this type from PlantingSitesList which may not be the best import path?

Reply:
Good point

Reply:
On second thought, I think this kinda makes sense. A child should be able to import types / utils / etc... from a parent. This is hyper specific to the view so it doesn't make sense to put it in the general planting sites types in my opinion

Reply:
Fair enough 👍